cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataUpgrader.java

```diff
@@ -45,7 +44,6 @@
   private final MetadataStore metadataStore;
   private final DatasetFramework datasetFramework;
 
-  @Inject
```

Comment:
Why removed?? I don't see any changes that do a `new MetadataUpgrader`.

Reply:
`MetadataUpgrader` is not used right now.